src/tasks/tasks.py

```diff
@@ -138,15 +139,13 @@ def count_examples(self, splits=['train', 'val', 'test']):
             count = self.get_num_examples(st)
             self.example_counts[split] = count
 
+    def tokenizer_is_supported(self, tokenizer_name):
+        ''' Check if the tokenizer is supported for this task. '''
+        return tokenizer_name == utils.TOKENIZER.__class__.__name__
```

Comment:
isn't this always `MosesTokenizer`? I'm not seeing where in utils this might change if you specify a different tokenizer.

Reply:
Yes; I'm just referencing that so it doesn't need to be defined in as many places.

Reply:
Gotcha. Just to check my understanding, a task with different tokenization should overwrite this to something like `tokenizer_name in [tokenizer1, tokenizer2, ...]`?

And once BERT is supported, this should look something like `tokenizer_name in [MosesTokenizer, BERTTokenizer]`?

Reply:
cool, thanks!